README.md

```diff
@@ -1,3 +1,5 @@
 ## Extra SM drivers for XCP-ng
 
 This repo contains a list of extra drivers for legacy XCP-ng storage stack.
+
+For XFS, you will need to install xfsprogs using ```# yum install xfsprogs --enablerepo=base```
```

Comment:
I have added xfsprogs to our updates repository and will add it as a dependency to sm-additional-drivers so this can be omitted.